### DISCUSSION TITLE
Add common terms by issues or modified directly from code?

### DISCUSSION THREAD

**Discussion by Kent-Liu** [openvanilla/McBopomofo#253] [General]
Dears,

I am trying to use it in daily using, it would be great if user-define terms feature could be come out.
But I also found some common terms might be useful to be added into current table,
Will you suggest me to modify them in code directly? or file a issue/suggestion for you guys to modify them in the future?
e.g.
捎帶 -> 稍待
時麼 -> 什麼
護理戰 -> 護理站
就款 -> 舊款
到或 -> 到貨
墊子黨 -> 電子檔
墊子 -> 電子
必司 -> 敝司
和加平安 -> 闔家平安
蒼屬 -> 倉鼠
林貴 -> 臨櫃
串街 -> 串接
... etc.

**Comment by lukhnos** [comment]
@Kent-Liu

> it would be great if user-define terms feature could be come out.

我們不久前發行了 2.0 版，用戶自訂增刪詞功能已經公開了。如果您已經是用 Xcode 編譯 git HEAD 的原始碼，該功能也已經啟用。

> I also found some common terms might be useful to be added into current table,
> Will you suggest me to modify them in code directly? or file a issue/suggestion for you guys to modify them in the future?

我們不久前更新了問題回報的模版，可以從那邊回報及討論詞庫問題: https://github.com/openvanilla/McBopomofo/issues/new/choose

> e.g.
> ```
> 捎帶 -> 稍待
> 時麼 -> 什麼
> 護理戰 -> 護理站
> […]
> 墊子 -> 電子
> ```

了解。這些目前可以透過用戶自訂詞來填補闕漏，但像「稍待」「護理站」等常用詞確實值得加入…

* 稍待：這像是缺詞
* 時麼：我猜應該是有些輸入法打ㄕˊ ㄇㄛ˙而非ㄕㄜˊ才能出「什麼」（否則通常出「甚麼」）
* 護理站：目前詞庫沒有這個詞，照讀音打預設出「護理戰」。現在有了選字記憶機制，至少對需要常打這個詞的使用者，可以減緩一些重複選字的次數。
* 墊子：小麥詞庫裡有「電子」但是子是三聲，輕聲的子出「墊子」

如果您目前已經有夠長的清單，請您[發個 bug](https://github.com/openvanilla/McBopomofo/issues/new/choose)給我們，讓我們有個更新依據。

謝謝！

**Reply by Kent-Liu** [reply]
@lukhnos Thanks for your reply, I will submit it latter.

**Comment by lukhnos** [comment]
> Will you suggest me to modify them in code directly?

另外，如果想透過修改資料檔增添讀音的話，可以參考這個 PR 的作法: https://github.com/openvanilla/McBopomofo/pull/161

增修詞彙的話，可參考: https://github.com/openvanilla/McBopomofo/pull/157

**Reply by Kent-Liu** [reply]
@lukhnos Thanks I may add some common term and submit as pull request.

**Comment by zonble** [comment]
另外也可以直接使用加詞表格，輸入這些

```
稍待 ㄕㄠ-ㄉㄞˋ
什麼 ㄕˊ-ㄇㄜ˙
護理站 ㄏㄨˋ-ㄌㄧˇ-ㄓㄢˋ
舊款 ㄐㄧㄡˋ-ㄎㄨㄢˇ
到貨 ㄉㄠˋ-ㄏㄨㄛˋ
電子檔 ㄉㄧㄢˋ-ㄗ˙-ㄉㄤˇ
電子 ㄉㄧㄢˋ-ㄗ˙
敝司 ㄅㄧˋ-ㄙ
闔家平安 ㄏㄜˊ-ㄐㄧㄚ-ㄆㄧㄥˊ-ㄢ
倉鼠 ㄘㄤ-ㄕㄨˇ
臨櫃 ㄌㄧㄣˊ-ㄍㄨㄟˋ
串接 ㄔㄨㄢˋ-ㄐㄧㄝ
```

然後按下 Reload user phrases 即可。

**Reply by Kent-Liu** [reply]
@zonble It is a great function! Thanks guys.

**Comment by mjhsieh** [comment]
Thanks for the suggestions and bug report, I will follow with some fix.
